buildtools/gradle/gradle_test.go

```diff
@@ -151,30 +152,6 @@ func MockGradle(t *testing.T, file string) gradle.ShellCommand {
 	}
 }
 
-func assertDosFile(t *testing.T, file []byte) {
-	fixture := string(file)
-	for i := range fixture {
-		if i == 0 {
-			continue
-		}
-		if fixture[i] == '\n' {
-			assert.Equal(t, uint8('\r'), fixture[i-1])
-		}
-	}
-}
-
-func assertUnixFile(t *testing.T, file []byte) {
-	fixture := string(file)
-	for i := range fixture {
-		if i == 0 {
-			continue
-		}
-		if fixture[i] == '\n' {
-			assert.NotEqual(t, uint8('\r'), fixture[i-1])
-		}
-	}
-}
-
 func findPackage(packages map[pkg.ID]pkg.Package, name, revision string) pkg.Package {
```

Comment:
you probably meant to delete this and `assertImport` from here

Reply:
Yea I'll kill this